sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -117,18 +117,20 @@ class AstBuilder extends DataTypeAstBuilder
   }
 
   override def visitCompoundOrSingleStatement(
-      ctx: CompoundOrSingleStatementContext): CompoundBody = withOrigin(ctx) {
+      ctx: CompoundOrSingleStatementContext): LogicalPlan = withOrigin(ctx) {
     Option(ctx.singleCompoundStatement()).map { s =>
+      if (conf.getConf(SQLConf.SQL_SCRIPTING_ENABLED)) {
```

Comment:
Why don't you use the active config object `conf.`? See other places in the file how to retrieve configs.

Reply:
From my understanding, I'm using active config object `.conf`. Am I correct?

Reply:
ah, sorry, I have an outdated branch locally.

Reply:
Done.